src/DocumentDB.ChangeFeedProcessor/Exceptions/LeaseLostException.cs

```diff
@@ -62,6 +50,19 @@ public LeaseLostException(string message, Exception innerException)
         {
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="LeaseLostException" /> class using the specified lease, inner exception, and a flag indicating whether lease is gone.
+        /// </summary>
+        /// <param name="lease">Instance of a lost lease.</param>
+        /// <param name="innerException">The inner exception.</param>
+        /// <param name="isGone">Whether lease doesn't exist.</param>
+        public LeaseLostException(ILease lease, Exception innerException, bool isGone)
```

Comment:
I would also remove unused ctors, wdyt?

Reply:
For this class I think it's OK. Or we can do that as separate change.